tests/bootstrap.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+declare(strict_types=1);
+
+use Doctrine\Deprecations\Deprecation;
+
+Deprecation::withoutDeduplication();
```

Comment:
FYI, I'm working on making this easier: https://github.com/doctrine/deprecations/pull/94